time/src/main/java/net/jqwik/time/api/Times.java

```diff
@@ -0,0 +1,70 @@
+package net.jqwik.time.api;
+
+import java.time.*;
+import java.util.*;
+
+import org.apiguardian.api.*;
+
+import net.jqwik.api.*;
+import net.jqwik.time.api.arbitraries.*;
+import net.jqwik.time.internal.properties.arbitraries.*;
+
+import static org.apiguardian.api.API.Status.*;
+
+@API(status = EXPERIMENTAL, since = "1.5.1")
+public class Times {
+
+	private Times() {
+		// Must never be called
+	}
+
+	/**
+	 * Create an arbitrary that generates instances of {@linkplain java.time.LocalTime}.
+	 *
+	 * @return a new arbitrary instance
+	 */
+	public static LocalTimeArbitrary times() {
+		return new DefaultLocalTimeArbitrary();
+	}
+
+	/**
+	 * Create an arbitrary that generates instances of {@linkplain java.time.OffsetTime}.
+	 *
+	 * @return a new arbitrary instance
+	 */
+	public static OffsetTimeArbitrary offsetTimes() {
+		return new DefaultOffsetTimeArbitrary();
+	}
+
+	public static ZoneOffsetArbitrary zoneOffsets() {
+		return new DefaultZoneOffsetArbitrary();
+	}
+
+	/**
+	 * Create an arbitrary that generates instances of {@linkplain java.util.TimeZone}.
+	 *
+	 * @return a new arbitrary instance
+	 */
+	public static Arbitrary<TimeZone> timeZones() {
```

Comment:
It could be derived from a ZoneId, couldn't it?

Reply:
I don't think so. 
`zoneIds()` generates 600 values,
`timeZones()` generates 628 values.